server/index-node/src/schema.graphql

```diff
@@ -7,6 +7,12 @@ scalar String
 type Query {
   indexingStatusesForSubgraphName(subgraphName: String!): [SubgraphIndexingStatus!]!
   indexingStatuses(subgraphs: [String!]): [SubgraphIndexingStatus!]!
+  proofOfIndexing(request: ProofOfIndexingInput!): Bytes
```

Comment:
It feels more natural _now_, but when there are some arbitrary count of block numbers because of subgraph composition over the availability chain and other non-Ethereum chains it's going to be weird that only the Ethereum chain is special in that it doesn't have a prefix for it's block number, and there are a bunch of optional arguments rather than optional fields on a struct.

I guess the future can wait though - especially since this is an internal API. Kind of hesitant about putting effort into changing it though.

Reply:
Yeah, understood. We can leave it the way it is.

Reply:
It's ok, I already changed it to your preference. This makes it the best code according to the current requirements.